### PR TITLE
Added some tests

### PR DESCRIPTION
**Pull request by joecohens**

### CODE REVIEW

File: src/NotifyMeFactory.php [hunk 1]
```diff
@@ -43,12 +43,26 @@ public function createFactory(array $config)
...
+    /**
+     * Resolve gateway class name from config.
+     *
+     * @param string $driver
+     *
+     * @return string
+     */
+    public function getDriverClassName($driver)
+    {
+        $className = ucfirst(strtolower($driver));
```
Comment (GrahamCampbell) [inline src/NotifyMeFactory.php]: Not correct I'm afraid. See my previous comment.

Comment (joecohens) [inline src/NotifyMeFactory.php]: Fixed merging :)

File: src/NotifyMeFactory.php [hunk 2]
```diff
@@ -43,12 +43,24 @@ public function createFactory(array $config)
...
     }
+
+    /**
+     * Resolve gateway class name from config.
+     *
+     * @param string $driver
+     *
+     * @return string
+     */
+    public function getDriverClassName($driver)
```
Comment (GrahamCampbell) [inline src/NotifyMeFactory.php:62]: This abstraction isn't required, and just adds an extra method call with no really benefit.

File: tests/NotifyMeFactoryTest.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     * @expectedExceptionMessage A driver must be specified.
+     */
+    public function testNoDriverSpecified()
+    {
+        $this->factory->make([]);
+    }
+
+    /**
+     * @expectedException InvalidArgumentException
+     * @expectedExceptionMessage Unsupported driver
```
Comment (GrahamCampbell) [inline tests/NotifyMeFactoryTest.php:35]: we should probably include the rest of the message in this test

File: src/NotifyMeFactory.php (hunk 2 above)
Comment (joecohens) [inline src/NotifyMeFactory.php:62]: Removing it

File: tests/NotifyMeFactoryTest.php (hunk 3 above)
Comment (joecohens) [inline tests/NotifyMeFactoryTest.php:35]: Sure

File: src/NotifyMeFactory.php (hunk 2 above)
Comment (joecohens) [inline src/NotifyMeFactory.php:62]: See https://github.com/notifymehq/notifyme/commit/069238d2647c61ae63113e6a17e4947fff3a17d7

**Pull request merged**